src/JsonPointer/RelativeJsonPointer.cs

```diff
@@ -122,7 +122,12 @@ public static RelativeJsonPointer Parse(string source)
 		while (i < span.Length && char.IsDigit(span[i])) i++;
 		if (i == 0) throw new PointerParseException($"`{nameof(source)}` must start with a non-negative integer");
 
+#if NET8_0_OR_GREATER
+		var parentSteps = uint.Parse(span[..i]);
+#else
 		var parentSteps = uint.Parse(span[..i].ToString());
+#endif
```

Comment:
Instead of repeating this, could you create a couple of extension methods in `UtilityExtensions`, please?  It reads better at the call site when there aren't compiler directives.

Reply:
Done